### PR TITLE
Makefile

### PR DESCRIPTION
**Pull request by georgebrock**
Wires together paperback and this repo's custom embedded script foo.

`make build` or just `make` will Take source material from `src/*.md`, run them through `bin/scriptrunner`, drop the output in `book/*.md`, and then invoke `paperback build`.

`make clean` will remove the build directory, and the generated source files in the book directory.

### CODE REVIEW

**Comment by calebhearth** [comment]
I think this needs a README update.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+SRCS=$(wildcard src/*.md)
+CHAPTERS=$(subst src,book,$(SRCS))
+
+.PHONY: clean
+
+build: $(CHAPTERS) .bundle
```
Comment (calebhearth) [inline Makefile]: Make has default tasks right? This seems like a good candidate.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+SRCS=$(wildcard src/*.md)
+CHAPTERS=$(subst src,book,$(SRCS))
+
+.PHONY: clean
```
Comment (calebhearth) [inline Makefile]: What's `.PHONY`?

File: Makefile (hunk 1 above)
Comment (georgebrock) [inline Makefile]: This is the default task: make takes the first one that doesn't start with a dot.

File: Makefile (hunk 2 above)
Comment (georgebrock) [inline Makefile]: PHONY tells make this target isn't a file, so it won't try to check if the file clean has changed

**Comment by georgebrock** [comment]
README updated, and I've made a few tweaks to the Makefile too.

**Comment by calleluks** [comment]
Looks good to me.

**Comment by mike-burns** [comment]
That `scriptrunner` is neat. Looks good to merge.

**Comment by georgebrock** [comment]
Thanks for the reviews. Merging.

**Pull request merged**